Investigate `ERROR: 'float' object cannot be interpreted as an integer` error

Since yesterday, all bindingTester's related CI seems broken. It might be related to https://forums.foundationdb.org/t/running-binding-tests/4639/2

https://github.com/foundationdb-rs/foundationdb-rs/actions/runs/12810557452/job/35717891584#step:8:20

```
Run scripts/run_bindingtester.sh 300
+ fdb_api_version=730
++ pwd
+ fdb_rs_dir=/home/runner/work/foundationdb-rs/foundationdb-rs
+ fdb_builddir=/home/runner/work/foundationdb-rs/foundationdb-rs/target/foundationdb_build
+ cd /home/runner/work/foundationdb-rs/foundationdb-rs/target/foundationdb_build/foundationdb
+ ./bindings/bindingtester/bindingtester.py --num-ops 1000 --api-version 730 --test-name api --compare python rust --seed 3534790651

Creating test at API version 730
Generating api test at seed 3534790651 with 1000 op(s) and 1 concurrent tester(s)...

ERROR: 'float' object cannot be interpreted as an integer
Error: Process completed with exit code 3.
```

--- a/.github/workflows/coverage.yml
+++ b/.github/workflows/coverage.yml
@@ -26,6 +26,12 @@ jobs:
         with:
           version: "7.3.27"
 
+      # Setting up a fixed Python version because of https://github.com/foundationdb-rs/foundationdb-rs/issues/226
+      - name: Setup fixed Python version
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
       - name: Enable tenant
         run: fdbcli --exec "configure single memory tenant_mode=optional_experimental"
 

--- a/.github/workflows/cron-correctness.yml
+++ b/.github/workflows/cron-correctness.yml
@@ -32,6 +32,12 @@ jobs:
         with:
           version: "7.3.27"
 
+      # Setting up a fixed Python version because of https://github.com/foundationdb-rs/foundationdb-rs/issues/226
+      - name: Setup fixed Python 3.11 version
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
       - name: Enable tenant
         run: fdbcli --exec "configure single memory tenant_mode=optional_experimental"
 

--- a/.github/workflows/pr-correctness.yml
+++ b/.github/workflows/pr-correctness.yml
@@ -30,6 +30,12 @@ jobs:
         with:
           version: "7.3.27"
 
+      # Setting up a fixed Python version because of https://github.com/foundationdb-rs/foundationdb-rs/issues/226
+      - name: Setup fixed Python version
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
       - name: Enable tenant
         run: fdbcli --exec "configure single memory tenant_mode=optional_experimental"
 

--- a/scripts/setup_bindingtester.sh
+++ b/scripts/setup_bindingtester.sh
@@ -9,6 +9,8 @@ pip install foundationdb==7.3.27
 fdboption_file="$(pip show foundationdb | grep Loca | awk '{print $2}')/fdb/fdboptions.py"
 echo "fdb option file: $fdboption_file"
 
+python3 --version
+
 ## build the python bindings
 (
   fdb_builddir=${fdb_rs_dir:?}/target/foundationdb_build

